packages/toolkit/README.md

```diff
@@ -568,6 +598,9 @@ config.plugins.push(
 module.exports = config;
 ```
 
+> [!NOTE]
+> When `useScriptModules` mode is enabled the config returned from webpack here changes from an object to an array of two objects. The first one is the scripts config which matches the traditional structure. And the second object is the config for the ESM instance.
```

Comment:
Can we add an example of how the config should be extended with this setting enabled?

Reply:
@nicholasio done in https://github.com/10up/10up-toolkit/pull/392/commits/1f2c329ed75d00f115934ada7cb31cf54a10661f